libraries/shared/watcher/storage_watcher.go

```diff
@@ -71,10 +78,28 @@ func (storageWatcher StorageWatcher) Execute(rows chan utils.StorageDiffRow, err
 	}
 }
 
+func (storageWatcher StorageWatcher) getTransformer(contractAddress common.Address) (transformer.StorageTransformer, bool) {
+	if storageWatcher.diffSource == "csv" {
+		storageTransformer, ok := storageWatcher.Transformers[contractAddress]
```

Comment:
yeah, i thought so too, but my IDE says `Not enough arguments to return` - I wonder if it's because getting a var from a map can return just the one value, and not the `ok` value